Resources/Maps/frontier.yml

```diff
@@ -37371,15 +37421,6 @@ entities:
     - pos: 25.5,18.5
       parent: 2173
       type: Transform
-- proto: VendingMachineSustenance
-  entities:
-  - uid: 6029
-    components:
-    - flags: SessionSpecific
-      type: MetaData
-    - pos: -15.5,12.5
-      parent: 2173
-      type: Transform
```

Comment:
While adding a restock box would technically be better, this solution suffices for now.

Reply:
We probably not going to just to avoid more food options that are not from food ships.